localtests/convert-utf8mb4/create.sql

```diff
@@ -7,7 +7,7 @@ create table gh_ost_test (
   primary key(id)
 ) auto_increment=1;
 
-insert into gh_ost_test values (null, 'átesting');
+insert into gh_ost_test values (null, 'átesting', null, null);
```

Comment:
Fixes `Testing: convert-utf8mb4..ERROR 1048 (23000) at line 10: Column 'tl' cannot be null`

Mismatch didn't seem intentional; I can revert if needed.